requirements.txt

```diff
@@ -1,10 +1,12 @@
 git+https://github.com/WIPACrepo/wipac-dev-tools@v1.0.11#egg=wipac_dev_tools
 coloredlogs==15.0.1
-opentelemetry-api==1.2.0
-opentelemetry-exporter-jaeger==1.2.0
-opentelemetry-exporter-otlp==1.2.0
+opentelemetry-api==1.5.0
+opentelemetry-exporter-jaeger==1.5.0
+opentelemetry-exporter-otlp==1.5.0
+opentelemetry-exporter-otlp-proto-grpc==1.5.0
+opentelemetry-exporter-otlp-proto-http==1.5.0
 opentelemetry-instrumentation-flask
 opentelemetry-instrumentation-requests
-opentelemetry-propagator-b3==1.2.0
-opentelemetry-sdk==1.2.0
+opentelemetry-propagator-b3==1.5.0
+opentelemetry-sdk==1.5.0
 pika==1.2.0
```

Comment:
I'm glad this updating seems to have worked. I had some issues last time with package compatibly during setup/install using the most recent versions.